data/clients.json

```diff
@@ -132,6 +132,10 @@
 			"name": "Apache HTTP Server",
 			"url": "https://httpd.apache.org"
 		}
```

Comment:
There should be a `,` here.

Reply:
Ha, yes, indeed, my bad. Thanks for spotting this :+1: